docs/user/forms.md

```diff
@@ -27,7 +30,38 @@ writer.update_page_form_field_values(
     writer.pages[0], {"fieldname": "some filled in text"}
 )
 
+# If you want to fill out *all* pages, it is also safe to do this:
+data = {"fieldname": "some filled in text", "othername": "more text for an input on a different page"}
+for page in writer.pages:
+    writer.update_page_form_field_values(page, data)
```

Comment:
can you have a look at #1903. more details about `auto_regenerate = False` may help many people

Reply:
I read over the linked thread and honestly I failed to fully understand it. At least, not well enough to be able to write any useful documentation about it. Someone more knowledgeable than I should probably provide those additional details as to why `auto_regenerate = False` is recommended.

Reply:
previously update_page_form_field_values() is computing the rendering of the field but also sets for the whole document the field `NeedAppearances` forcing the viewer to recompute the rendering of the fields. the side effect is that the document will be marked as modified and the user will be ask to save the file. if you set the auto_regenerate parameter to false, NeedAppearances will be set to false. 
the default value of value of auto_regenerate is true to keep compatibility with legacy behavior.